---
permalink: /
title: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Hi, I'm Xiaoke, a PhD candidate in Department of pharmaceutical science, University of Nebraska Medical Center

Summary
======
pharmaceutical science. Has both wet lab and dry lab experience and skills. My thesis project focused on using
scRNA-seq and scATAC-seq to understand HIV-associated neurological disorders (HANDs). I was also involved in
several other collaborated studies using bulk RNA-seq and proteomics data to understand other neuropathologies. I
am skilled in R and Python programming languages, Linux bash, and varied bioinformatic/analytical tools.

Education
======
China Pharmaceutical University (09/2016-06/2020)
------
* Bachelor of Science in Pharmaceutical Science (GPA 3.6/4.0)
* Award: Chinese government scholarship; University scholarship(3 years)

University of Nebraska Medical Center (08/2021-08/2025 (expected))
------
* PhD in Pharmaceutical Science (GPA 3.9/4.0)
* Selected courses: Biostatistical Computing (A+), Intro to Biostatistics, Machine Learning (A+), Biostatistics (A), Fundamentals of Biomolecules(A-), Intro (A+) and Advanced (A) Immunology
* Leadership position: resident of China Students and Scholars Association

Research
======
1. Unveiling the transcriptomic landscape of brain immunity in acute SIV infection via single-cell RNA sequencing (scRNA-seq). 
  In this project, the scRNA-seq was the main technique used. My work includes [preprocessing the scRNA-seq data](https://www.10xgenomics.com/support/software/cell-ranger/latest), data mining and interpretation. In data mining, I used the [Seurat R pacakge](https://satijalab.org/seurat/). Additionally, other scRNA-seq analysis, data manipulation, and visualization tools were also used, which includes [clusterProfiler](https://github.com/YuLab-SMU/clusterProfiler), [ComplexHeatmap](https://github.com/jokergoo/ComplexHeatmap), [Monocle3](https://cole-trapnell-lab.github.io/monocle3/), [dplyr](https://github.com/tidyverse/dplyr), and [ggplot2](https://ggplot2.tidyverse.org/). Given the advancement of scRNA-seq in studying the disease mechanism, I had serveral interesting findings. In the study of [Microglia and macrophages alterations in the CNS during acute SIV infection](https://doi.org/10.1371/journal.ppat.1012168), I identified signature genes associated with different phenotypes and mapping them to various biological and pathological pathways. I also discovered two myeloid cell clusters strongly linked to neurodegenerative disorders. These findings might shed light on the immunological and pathological effects of different myeloid phenotypes in the brain during acute SIV infection, providing valuable insights for future therapeutic strategies targeting this critical stage and aiming to eliminate the viral reservoir. In the study of Lymphocyte alternations in the CNS during acute SIV infection (paper in preparation), I did a comprehensive transcriptomic characterization of brain lymphocytes and blood CD4+ T cells during acute SIV infection, identifying a candidate lymphocyte phenotype that may contribute to the initiation of CNS infection. 
